docs/client/data.js

```diff
@@ -5716,7 +5716,17 @@ DocsData = {
     "memberof": "Mongo.Cursor",
     "name": "count",
     "options": [],
-    "params": [],
+    "params": [
```

Comment:
Actually, these docs are generated from JSDoc in the source code. See the first line in this file for instructions.